src/components/NumberInput.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+
+import React, {FC} from 'react';
+import {ErrorMessage, FieldProps, useFormikContext} from 'formik';
+import {CiCircleMinus, CiCirclePlus} from 'react-icons/ci';
+
+interface NumberInputWithButtonsProps extends FieldProps {
+  minValue?: number;
+  maxValue?: number;
+}
+
+const NumberInputWithButtons: FC<NumberInputWithButtonsProps> = ({
+  field,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  form, // Needed for 'field'-element to work
+  minValue,
+  maxValue,
+  ...props
+}) => {
+  const { setFieldValue, getFieldProps, setFieldTouched } = useFormikContext<{ [key: string]: number }>();
+  const [showCustomText, setShowCustomText] = React.useState<string | undefined>(undefined);
+
+  const increaseValue = () => {
+    const oldValue = +getFieldProps(field.name).value;
+    if (maxValue === undefined || oldValue < maxValue) {
+      void setFieldTouched(field.name, true);
+      void setFieldValue(field.name, oldValue + 1);
+    } else if (oldValue >= maxValue) {
+      setShowCustomText(`Maksverdi nådd (${maxValue})`);
+      setTimeout(() => setShowCustomText(undefined), 2000);
+    }
+  };
+
+  const decreaseValue = () => {
+    const oldValue = +getFieldProps(field.name).value;
+
+    if (minValue === undefined || oldValue > minValue) {
+      void setFieldTouched(field.name, true);
+      void setFieldValue(field.name, oldValue - 1);
+    } else if (oldValue <= minValue) {
+      setShowCustomText(`Minimumsverdi nådd (${minValue})`);
+      setTimeout(() => setShowCustomText(undefined), 2000);
+    }
+  };
+
+  return (
+    <div>
+      <div className='flex flex-row'>
+        <button
+          type='button'
+          onClick={increaseValue}
+        >
+          <CiCirclePlus size={30} style={{color: 'green'}}/>
+        </button>
+
+        <input
+          type='number'
+          {...field}
+          {...props}
+        />
+
+        <button
+          type={'button'}
+          onClick={decreaseValue}
+        >
+          <CiCircleMinus size={30} style={{color: 'red'}}/>
```

Comment:
Føles litt kontraintuitivt for meg å ha + til venstre og - til høyre, men det var kanskje gjennomtenkt?

Reply:
Hmm er sånn vi har det på relation, har egentlig ikke tenkt noe om det. Skal høre med Mona kjapt.

Reply:
Mona var enig med deg, så har byttet om!